diff --git a/dist/src/stryker-setup.js b/dist/src/stryker-setup.js
index aecfacbe47b285c272c47e54b3d66a716adfdc41..d7ac0c5d706313c6b472cde8f59677698ca59e32 100644
--- a/dist/src/stryker-setup.js
+++ b/dist/src/stryker-setup.js
@@ -63,7 +63,7 @@ function collectTestName({ name, suite, }) {
         nameParts.unshift(currentSuite.name);
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(' > ').trim();
 }
 function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
diff --git a/dist/src/test-helpers.js b/dist/src/test-helpers.js
index 56cc786f30c524ae18ce87eef1a67fdf26b703c5..2cd9786a153a6b3090b84e30e02750d1c20b5a0b 100644
--- a/dist/src/test-helpers.js
+++ b/dist/src/test-helpers.js
@@ -8,7 +8,7 @@ export function collectTestName({ name, suite, }) {
         nameParts.unshift(currentSuite.name);
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(' > ').trim();
 }
 export function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
